### PR TITLE
Add Linxdot Hotspot

### PR DESCRIPTION
**Pull request by eriks12**
This is to add the Linxdot to the hotspot app. The only thing that needs changing is the onboarding ID in src/makers/linxdot/index.ts, which I'll be in talks with DeWi for. Please let me know if any changes are needed.

### CODE REVIEW

File: src/makers/linxdot/hotspots.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import ANTENNAS from './antennas'
+
+const Linxdot = {
+  name: 'Linxdot Hotspot',
+  icon: HotspotIcon,
+  onboardType: 'BLE',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Linxdot to identify issues with your Hotspot in a secure way.</white></b>\n\nLinxdotwill never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@linxdot.com</b></purple> from the email used to purchase the Hotspot.',
```
Comment (cokes518) [inline src/makers/linxdot/hotspots.ts:12]:
```suggestion
        '<b><white>Diagnostic support allows Linxdot to identify issues with your Hotspot in a secure way.</white></b>\n\nLinxdot will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@linxdot.com</b></purple> from the email used to purchase the Hotspot.',
```

**Review by cokes518 — CHANGES_REQUESTED** [review]

**Comment by cokes518** [comment]
this is good to merge (copy-wise)

**Comment by cokes518** [comment]
cc @tyler-whitman this is good to merge as well.

File: src/makers/linxdot/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 99999, // TODO: Get maker id from onboarding server
```
Comment (tyler-whitman) [inline src/makers/linxdot/index.ts:7]: Pending approved maker id

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

**Comment by cokes518** [comment]
Hi @eriks12 as we await the MOC approval, please let me know the name you'd like to appear in the Onboarding server, and how many location asserts you'd like to include (minimum is 1).

Here is an example of other Makers and the names they use.
https://onboarding.dewi.org/api/v2/makers

**Comment by eriks12** [comment]
> Hi @eriks12 as we await the MOC approval, please let me know the name you'd like to appear in the Onboarding server, and how many location asserts you'd like to include (minimum is 1).
> 
> Here is an example of other Makers and the names they use.
> https://onboarding.dewi.org/api/v2/makers

Hi, thanks for getting in touch! We’re happy with “Linxdot” as the name on the onboarding server. 10 location asserts is fine to begin with, and we’ll fund our maker account additionally once it’s created. 

Many thanks.
-Erik

**Comment by cokes518** [comment]
Hi Erik, are you sure you want to pay for each user's assert_location transactions, up to 10? That's $100 in assert fees per user.

**Comment by eriks12** [comment]
> Hi Erik, are you sure you want to pay for each user's assert_location transactions, up to 10? That's $100 in assert fees per user.

Sorry, I misunderstood. 1 included location assert only :).

I thought this was about initial funding of a maker account.

File: src/makers/linxdot/index.ts (hunk 2 above)
Comment (cokes518) [inline src/makers/linxdot/index.ts:7]:
```suggestion
  id: 15,
```

**Review by cokes518 — APPROVED** [review]
maker account created
```{
    "code": 200,
    "data": {
        "id": 15,
        "name": "Linxdot",
        "address": "14eUfY1GsjK4WH6uZYoeagnFtigBKdvPruAXLmc5UsUMEDj3yib",
        "locationNonceLimit": 1
    },
    "success": true
}```

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**